<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSelectorTreeViewWidget</class>
 <widget class="QWidget" name="NodeSelectorTreeViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">#Container
{
background-color: #FFFFFF; /*colorToken.page-background*/
}

NodeSelectorTreeView
{
background-color: #FFFFFF; /*colorToken.page-background*/
color: #303233; /*colorToken.text-primary*/
border: none;
}

NodeSelectorTreeView::item::selected
{
background-color: transparent;
}


#searchEmptyInfoWidget
{
color:#3D3D3D;
font-size:16px;
font-weight: 700;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <item row="0" column="8">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="6">
         <spacer name="avatarSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QToolButton" name="lSyncIcon">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="class" stdset="0">
           <string notr="true">icon-label</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>12</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3" colspan="10" alignment="Qt::AlignLeft">
         <widget class="QWidget" name="searchButtonsWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="TabSelector" name="cloudDriveSearch" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="class" stdset="0">
              <string notr="true">chip</string>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../../../gui/Resources_common.qrc">
               <normaloff>:/cloud_off</normaloff>
               <normalon>:/cloud_on</normalon>:/cloud_off</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TabSelector" name="backupsSearch" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="class" stdset="0">
              <string notr="true">chip</string>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../../../gui/Resources_common.qrc">
               <normaloff>:/devices_off</normaloff>
               <normalon>:/devices_on</normalon>:/devices_off</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TabSelector" name="incomingSharesSearch" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="class" stdset="0">
              <string notr="true">chip</string>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../../../gui/Resources_common.qrc">
               <normaloff>:/folder-users_off</normaloff>
               <normalon>:/folder-users_on</normalon>:/folder-users_off</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TabSelector" name="rubbishSearch" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="class" stdset="0">
              <string notr="true">chip</string>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../../../gui/Resources_common.qrc">
               <normaloff>:/search_outline_small</normaloff>
               <normalon>:/search_solid_small</normalon>:/search_outline_small</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="lFolderName">
          <property name="font">
           <font>
            <bold>true</bold>
            <strikeout>false</strikeout>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="text">
           <string notr="true">FolderName</string>
          </property>
          <property name="font-size" stdset="0">
           <string notr="true">body-2</string>
          </property>
          <property name="bold" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="lAccess">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="IconOnlyButton" name="bBack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../gui/Resources_common.qrc">
            <normaloff>:/chevron-left_medium_thin_outline</normaloff>:/chevron-left_medium_thin_outline</iconset>
          </property>
          <property name="dimension" stdset="0">
           <string notr="true">medium</string>
          </property>
          <property name="type" stdset="0">
           <string notr="true">ghost</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QToolButton" name="lOwnerIcon">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="class" stdset="0">
           <string notr="true">icon-label</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="IconOnlyButton" name="bForward">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../gui/Resources_common.qrc">
            <normaloff>:/chevron-right_medium_thin_outline</normaloff>:/chevron-right_medium_thin_outline</iconset>
          </property>
          <property name="dimension" stdset="0">
           <string notr="true">medium</string>
          </property>
          <property name="type" stdset="0">
           <string notr="true">ghost</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="syncSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="10">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="11">
         <widget class="QWidget" name="customButtonsContainer" native="true">
          <layout class="QHBoxLayout" name="customButtonsLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="treeViewPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="NodeSelectorTreeView" name="tMegaFolders">
            <property name="font">
             <font>
              <family>SF Pro Display</family>
              <bold>false</bold>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoExpandDelay">
             <number>-1</number>
            </property>
            <property name="indentation">
             <number>18</number>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>false</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <attribute name="headerMinimumSectionSize">
             <number>35</number>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>35</number>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="emptyPage">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
              <widget class="QToolButton" name="emptyIcon">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="../../../gui/Resources_common.qrc">
                 <normaloff>:/images/node_selector/view/cloud.png</normaloff>:/images/node_selector/view/cloud.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QWidget" name="searchEmptyInfoWidget" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="textLabel">
                  <property name="text">
                   <string>No search results</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="footer" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>21</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>21</number>
         </property>
         <item>
          <widget class="QCheckBox" name="cbAlwaysUploadToLocation">
           <property name="text">
            <string>Always upload to this destination</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="bLayout" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="bNewFolder">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>New folder</string>
              </property>
              <property name="icon">
               <iconset resource="../../../gui/Resources_common.qrc">
                <normaloff>:/images/node_selector/plus.png</normaloff>:/images/node_selector/plus.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="dimension" stdset="0">
               <string notr="true">small</string>
              </property>
              <property name="font-size" stdset="0">
               <string notr="true">body-2</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="bCancel">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bOk">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Ok</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabSelector</class>
   <extends>QWidget</extends>
   <header>TabSelector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconOnlyButton</class>
   <extends>QPushButton</extends>
   <header>TokenizableItems/TokenizableButtons.h</header>
  </customwidget>
  <customwidget>
   <class>NodeSelectorTreeView</class>
   <extends>QTreeView</extends>
   <header>node_selector/gui/NodeSelectorTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../gui/Resources_common.qrc"/>
 </resources>
 <connections/>
</ui>
